### PR TITLE
Add update method

### PR DESCRIPTION
**Pull request by javierg**

### CODE REVIEW

File: lib/dolly/document.rb [hunk 1]
```diff
@@ -132,5 +144,9 @@ def init_doc options
       self.doc['_id'] = self.class.namespace( normalized_id ) if normalized_id
     end
 
+    def valid_properties(options)
+      options.keys.any?{ |option| _properties.map(&:name).include?(option.to_s) }
```
Comment (rubenrails) [inline lib/dolly/document.rb]: Will this allow only updating existing properties on a document? or will this save even new added properties on update?

Comment (javierg) [inline lib/dolly/document.rb]: it allow new attributes if they are defined with `property :attribute`

File: lib/dolly/document.rb [hunk 2]
```diff
@@ -132,5 +144,9 @@ def init_doc options
       self.doc['_id'] = self.class.namespace( normalized_id ) if normalized_id
     end
 
+    def valid_properties(options)
```
Comment (seancookr) [inline lib/dolly/document.rb]: this method should have a question mark at the end since it returns a boolean value

**Comment by javierg** [comment]
@seancookr @rubenrails can I merge this?

**Comment by seancookr** [comment]
Looks good to me.

**Pull request merged**